A28-xds-traffic-splitting-and-routing.md

```diff
@@ -369,7 +363,18 @@ field](https://github.com/envoyproxy/envoy/blob/v1.13.1/api/envoy/api/v2/route/r
     [query\_parameters](https://github.com/envoyproxy/envoy/blob/v1.13.1/api/envoy/api/v2/route/route_components.proto#L442)
     *   Note: the entire route containing this will be ignored, regardless of
         the other matchers, this is the same as this always evaluates to false,
-        because gRPC does not and will not support query parameters)
+        because gRPC does not and will not support query parameters
+*   Can have
+    [grpc](https://github.com/envoyproxy/envoy/blob/v1.13.1/api/envoy/api/v2/route/route_components.proto#L447)
+    *   This matcher will be ignored (but the other matchers in the route will
+        be evaluated). The `content-type` can be used instead to filter gRPC
```

Comment:
I don't think this last sentence is necessary, because there is no case in which users would need to do this.  In gRPC, all requests will have the same content type, so there's no need for any filtering here.

Reply:
Removed.